poprox-db/migrations/versions/2024_10_29_1406-f8667df4b049_merge_head_revisions.py

```diff
@@ -0,0 +1,26 @@
+"""merge head revisions
+
+Revision ID: f8667df4b049
+Revises: 0f5739c4d79d, 9bd2ccb216a6
+Create Date: 2024-10-29 14:06:37.323505
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
```

Comment:
we can probably get this fixed when we want to merge. Manual fixing these isn't too hard, but I think `pre-commit run --all-files` or something similar should also fix the problem.